app/Http/Controllers/QuizApiController.php

```diff
@@ -34,5 +34,26 @@ public function find(int $id): JsonResponse
         ]);
     }
 
+    public function create(Request $request): JsonResponse
+    {
+        $request->validate([
+            'name' => 'required|string|min:1',
+            'description' => 'required|string|min:1',
+        ]);
```

Comment:
Suggest you move this validation into a request object - you'll probably need exactly the same validation for the edit quiz story, so having a request object would allow you to reuse that later.